src/handlers.cpp

```diff
@@ -1670,6 +1698,7 @@ void RegistrationTerminationTask::run()
     SAS::Event event(this->trail(), SASEvent::INVALID_DEREG_REASON, 0);
     SAS::report_event(event);
     send_rta(DIAMETER_REQ_FAILURE);
+    rtr_results_tbl->increment(SNMP::DiameterAppId::BASE, 5012);
```

Comment:
It's a bit of a shame that we have the string "DIAMETER_UNABLE_TO_COMPLY" and then have to separately have the constant 5012. One way to make this a bit nicer would be to move this increment inside send_rta() - it can then check its string argument and update the table appropriately.